.env.app

```diff
@@ -18,6 +18,8 @@ REACT_APP_UNCHAINED_ETHEREUM_HTTP_URL=https://api.ethereum.shapeshift.com
 REACT_APP_UNCHAINED_ETHEREUM_WS_URL=wss://api.ethereum.shapeshift.com
 REACT_APP_UNCHAINED_AVALANCHE_HTTP_URL=https://api.avalanche.shapeshift.com
 REACT_APP_UNCHAINED_AVALANCHE_WS_URL=wss://api.avalanche.shapeshift.com
+REACT_APP_UNCHAINED_OPTIMISM_HTTP_URL=https://dev-api.optimism.shapeshift.com
```

Comment:
If we are using the same URL for all (or most environments) do we just want to use `.env.base`?

Reply:
it won't be that way long term, just need to stand up prod coinstacks still